src/redis-cli.c

```diff
@@ -1430,6 +1430,10 @@ static void repl(void) {
                 } else if (argc == 1 && !strcasecmp(argv[0],"clear")) {
                     linenoiseClearScreen();
                 } else {
+                    if (!strcasecmp(argv[0],"flushall")) {
```

Comment:
I would also say we should add flushdb here as well, since for many workloads these are basically the same command.